debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TablesWithoutPrimaryKeyIT.java

```diff
@@ -62,6 +84,29 @@ public void shouldProcessFromSnapshot() throws Exception {
 
     @Test
     public void shouldProcessFromStreaming() throws Exception {
+        start(PostgresConnector.class, TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.SNAPSHOT_MODE, SnapshotMode.NEVER)
+                .with(PostgresConnectorConfig.SCHEMA_INCLUDE_LIST, "nopk")
+                .build());
+        assertConnectorIsRunning();
+        waitForStreamingToStart();
+
+        TestHelper.execute(STATEMENTS);
+
+        final int expectedRecordsCount = 1 + 1 + 1;
+
+        TestConsumer consumer = testConsumer(expectedRecordsCount, "nopk");
+        consumer.await(TestHelper.waitTimeForRecords(), TimeUnit.SECONDS);
+        final Map<String, List<SourceRecord>> recordsByTopic = recordsByTopic(expectedRecordsCount, consumer);
+        Assertions.assertThat(recordsByTopic.get("test_server.nopk.t1").get(0).keySchema().field("pk")).isNotNull();
```

Comment:
Would still prefer the static import.

Reply:
I logged https://issues.redhat.com/browse/DBZ-2432 to change it everywhere, not covering it in this PR